src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/pybossa/PyBossaPlatform.java

```diff
@@ -134,6 +141,9 @@ public Boolean isCalibrationAllowed() {
      * @throws UnidentifiedWorkerException if the worker cannot be identified
      */
     private String identifyWorker(Map<String, String[]> param) throws UnidentifiedWorkerException {
+        if (param == null) {
+            return "";
```

Comment:
the problem is how we know determine whether the platform supports the getWorker method. The Platform-manager calls getWorker(null) and if the result is not empty it assumes the platform supports identifying workers. It is documented in the interface. when the rest-call calls the method the map will be empty, not null.